build.gradle

```diff
@@ -83,6 +83,9 @@ subprojects { subproject ->
 		springRetryVersion = '1.2.2.RELEASE'
 		springVersion = '5.1.0.BUILD-SNAPSHOT'
 		springDataCommonsVersion = '2.1.0.BUILD-SNAPSHOT'
+		reactorCoreVersion = '3.2.0.M4'
+		reactorKafkaVersion = '1.0.1.RELEASE'
+		reactorTestVersion = '3.2.0.M4'
```

Comment:
This is redundant property. We are fully fine just use a single `reactorVersion = '3.2.0.M4'`

Reply:
Done